### PR TITLE
Fix testnet issues

### PR DESCRIPTION
**Pull request by shrinathprabhu**
Resolves [AR-3166](https://team-1624093970686.atlassian.net/browse/AR-3166).

## Changes
- Replace links with beta links
- change `magic link` to `login link`
- Make header sticky
- Change Arcana Wallet sdk reference to Arcana Auth
- Increase margin between content and full width image

## Checklist

- [x] The branch name follows the format: `developer/AR-XXX-issue-name`.
- [x] The changes have been tested locally.

### CODE REVIEW

File: components/OverviewSection.vue [hunk 1]
```diff
@@ -1,38 +1,18 @@
...
           Overview
         </v-heading>
         <div>
-          <v-text
-            class="overview-description"
-            color="secondary"
-            :weight="400"
-            line-height="1.4"
-          >
+          <v-text class="overview-description" color="secondary" :weight="400" line-height="1.4">
```
Comment (karthik-durai) [inline components/OverviewSection.vue]: line-height is very specific, can we change it to `1.5`

File: components/TheHeader.vue [hunk 2]
```diff
@@ -29,7 +30,37 @@ export default {
...
+    const headerInstance = this;
+    headerInstance.checkIsScrollAtTop()
+
+    window.onscroll = function (ev) {
+      headerInstance.checkIsScrollAtTop()
+    }
+  },
+  methods: {
+    checkIsScrollAtTop() {
+      if (window.scrollY < 50) {
```
Comment (karthik-durai) [inline components/TheHeader.vue]:
```suggestion
this.isScrollAtTop = window.scrollY < 50
```

**Review by karthik-durai — APPROVED** [review]

File: components/TheHeader.vue (hunk 2 above)
Comment (shrinathprabhu) [reply components/TheHeader.vue]: Resolved at: https://github.com/arcana-network/testnet-website/pull/23/commits/a0122211d53fc084ebfe3d33fde649e5b2a399ca

File: components/OverviewSection.vue (hunk 1 above)
Comment (shrinathprabhu) [reply components/OverviewSection.vue]: Resolved at: https://github.com/arcana-network/testnet-website/pull/23/commits/28faa2a5f380dd3b85649a91b750d7b3bf0b18f1

**Pull request merged**